syntax = "proto3";

message MxpiMetaHeader
{
    string parentName = 1;
    int32 memberId = 2;
    string dataSource = 3;
}

message MxpiWebDisplayDataList
{
    repeated MxpiWebDisplayData webDisplayDataVec = 1;
}

message MxpiWebDisplayData
{
    repeated MxpiMetaHeader headerVec = 1;
    bytes h264_data = 2;
    uint32 h264_size = 3;
    repeated MxpiBBox bbox_vec = 4;
    string channel_id = 5;
    uint32 frame_index = 6;
}

message MxpiBBox
{
    float x0 = 1;
    float y0 = 2;
    float x1 = 3;
    float y1 = 4;
}